libs/openFrameworks/gl/ofShader.cpp

```diff
@@ -1070,6 +1077,151 @@ void ofShader::setUniforms(const ofParameterGroup & parameters) const{
 		}
 	}
 }
+
+
+//--------------------------------------------------------------
+template<typename T>
+void ofShader::setDefineConstantTemp(const string & name, T value){
+#if (!defined(TARGET_LINUX) || defined(GCC_HAS_REGEX))
```

Comment:
yes it will, it only won't work on old gccs, anything below 4.9 which almost nobody should be using by now, i was planning to add an error message in case it's not implemented though but completely forgot